#!/usr/bin/env python
PACKAGE = "c_tracking"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#Extraction parameters
gen.add("extraction_threshold", int_t, 0, "Threshold for feature extraction", 10,  0, 100)

#Bounding box parameter
gen.add("boundingBox_xScaling", double_t, 0, "Bounding box initialization scaling on the x direction", 1.1, 1.0, 1.5)
gen.add("boundingBox_yScaling", double_t, 0, "Bounding box initialization scaling on the y direction", 1.1, 1.0, 1.5)

exit(gen.generate(PACKAGE, "c_tracking", "Parameters"))